account_cost_center/views/account_cost_center.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<openerp>
+    <data>
+
+        <!-- Cost Center views -->
+        <record id="account_cost_center_form" model="ir.ui.view">
+            <field name="name">account.cost.center.form</field>
+            <field name="model">account.cost.center</field>
+            <field name="arch" type="xml">
+                <form string="Cost Center" version="7.0">
+                    <sheet>
+                        <group>
+                            <div class="oe_title">
+                                <div class="oe_edit_only">
+                                    <label for="name" string="Title"/>
+                                </div>
+                                <h1>
+                                    <field name="name" required="1"/>
+                                </h1>
+                            </div>
+                        </group>
+                        <group name="main_group">
+                            <group name="main_group1">
+                                <field name="code"/>
+                            </group>
+                            <group name="main_group2">
+                            </group>
+                        </group>
+                        <notebook name="notebook">
+                        </notebook>
```

Comment:
This was meant to be used by modules that extend this one.
I was creating some new modules that add pages on this form and I thought it was useful to have a notebook here.
Let say we remove this notebook, then if you have other two modules that add pages, you will need to couple those modules: one module adds the notebook, the other module extends from the first one adding pages.

I would prefer to keep this empty elements here in order to make inheritance easier.